content/_index.html

```diff
@@ -0,0 +1,127 @@
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Bosch</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Cybernet Systems</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Dassault Systèmes</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">DLR</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">dSpace</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Esterel Technologies</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">ETAS</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">eXXcellent solutions</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Global Crown</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Honda</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">ITI</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Maplesoft</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Modelon</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">OSMC</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">PMSF</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Pratt &amp; Miller Engineering</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Siemens</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Simpack</div></div>
+            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Synopsis</div></div>
```

Comment:
```suggestion
            <div class="col"><div class="d-flex justify-content-center align-items-center bg-light rounded-3" style="height: 3rem;">Synopsys</div></div>
```

Either fix here, accept #15 first or let #15 resolve the conflict after this PR got merged.